factory/README.md

```diff
@@ -0,0 +1,172 @@
+# cypress/factory
+
+cypress/factory is a docker container that can be used with `args` to generate a docker container with specific versions of:
+
+* node
+* yarn
```

Comment:
always both or could it be one or the other?

Reply:
always node, yarn is optional